ion-schema/src/model/type_reference.rs

```diff
@@ -35,20 +42,18 @@ impl TypeReference {
         self.type_name.as_str()
     }
 
-    /// Returns an immutable borrow of the [TypeDefinition] to which this [TypeReference] refers.
-    pub fn get(&self) -> Option<&TypeDefinition> {
-        todo!()
+    pub(crate) fn type_coordinates(&self) -> Option<TypeCoordinates> {
+        self.resolved_type_coordinates
     }
 
-    // TODO: functions for setting the resolved type
+    pub(crate) fn resolve_type_coordinates(&mut self, type_coordinates: Option<TypeCoordinates>) {
```

Comment:
I think I am missing context on what this function is doing. IS this placing a resolved type? If so, why does it take an `Option`.

Reply:
Changed it to `set_type_coordinates` so that it doesn't look confusing that it can accept `None`.